### DISCUSSION TITLE
Return config to before snacks.nvim

### DISCUSSION THREAD

**Discussion by queirozlc** [LazyVim/LazyVim#4794] [Q&A] [answered]
Hello guys, i think this might be the recent bigger feature in Lazyvim project, but with all due respect with you guys who actively contribute with the project, i think i like more how it was before, it's possible to return my config as they were before ? At least disable the notifier, which is the thing that bothers me most

**Comment by dpetka2001** [comment, marked as answer]
You can disable the notifier in the `snacks.nvim` configuration in your personal files.
